src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -1501,3 +1501,89 @@ ActStatus ActionHandlers::handle_s1_rel_req_during_attach(ControlBlock& cb)
 
     return ActStatus::PROCEED;
 }
+
+#define S10_FEATURE
+
+#ifdef S10_FEATURE
+
+/***************************************
+* Action handler : send_identification_request_to_old_mme
+***************************************/
+ActStatus ActionHandlers::send_identification_request_to_old_mme(ControlBlock& cb)
+{
+	log_msg(LOG_DEBUG, "Inside send_identification_request_to_old_mme");
+
+	UEContext *ue_ctxt = dynamic_cast<UEContext*>(cb.getPermDataBlock());
+	VERIFY_UE(cb, ue_ctxt, "Invalid UE");
+
+	struct ID_Q_msg id_msg;
+	id_msg.msg_type = identification_request;
+	id_msg.ue_idx = ue_ctxt->getContextID();
+
+	memcpy(&(id_msg._guti.plmn_id), &(ue_ctxt->getTai().tai_m.plmn_id), 3);
+	id_msg._guti.mme_grp_id = htons(mme_cfg->mme_group_id);
```

Comment:
Why variable name _guti starts with _ ?

Reply:
Just to differentiate with structure name. We can replace with guti itself.